build-clang

```diff
@@ -485,7 +485,7 @@ function package_tc() {
             *)
                 die "Invalid compression specified... skipping" ;;
         esac
-        find "${DIR}" -type f -name "${TARGET_PACKAGE}" | xargs mv -t "${TC_FOLDER}"
+        find "${DIR}" -print0 -type f -name "${TARGET_PACKAGE}" | xargs mv -t "${TC_FOLDER}"
```

Comment:
```
xargs: Warning: a NUL character occurred in the input.  It cannot be passed through in the argument list.  Did you mean to use the --null option?
xargs: argument line too long
```
This should be like this
`find "${DIR}" -print0 -type f -name "${TARGET_PACKAGE}" | xargs --null mv -t "${TC_FOLDER}"`

Reply:
Done.